### ISSUE TITLE
Support for design document update handlers?

### ISSUE THREAD

**Issue by geekytime** [chris-l/mock-couch#34]
Thanks for the great library. I've been looking for something like this for a while!

The docs clearly show that views are supported using addDoc, but it doesn't appear that `updates` functions are supported when I add design docs that have update handlers in them. We use update handlers to add simple timestamps to our docs. Am I doing something wrong, or is this feature just not supported yet.

Are update handlers a feature that you'd consider supporting, if they don't work already?

**Comment by chris-l** [comment]
Yes, you are right. This is something Mock-Couch is currently missing and that is important to have. I have started to work on this, so I hope is going to be ready soon.

**Comment by 0x1mason** [comment]
+1

**Comment by sagivmalihi** [comment]
Also in need of this feature. Can I help maybe?

**Comment by 0x1mason** [comment]
I ended up wrapping PouchDb. A similar approach would take a lot of the implementation pressure off and just allow you to focus in the API.
